docs/source/usage/using_vhdl.md

```diff
@@ -1,24 +1,42 @@
 # Using VHDL
 
-LibreLane supports VHDL by using the GHDL plugin for Yosys.
+LibreLane supports VHDL **only on x86-64 platforms** by using the GHDL plugin
```

Comment:
Not anymore, thanks to https://github.com/fossi-foundation/nix-eda/pull/32

Do you want to update nix-eda in this PR as well?

Reply:
Ah, right... this goes to main!